<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGuiApplicationClass</class>
 <widget class="QMainWindow" name="QtGuiApplicationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>947</width>
    <height>783</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGuiApplication</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>21</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <widget class="QToolButton" name="addPipelineToolButton">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>21</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="QtGuiApplication.qrc">
             <normaloff>:/png/add.png</normaloff>:/png/add.png</iconset>
           </property>
          </widget>
          <widget class="QToolButton" name="removePipelineToolButton">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>0</y>
             <width>21</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="QtGuiApplication.qrc">
             <normaloff>:/png/delete.png</normaloff>:/png/delete.png</iconset>
           </property>
          </widget>
          <widget class="QToolButton" name="explorerPipelineFileToolButton">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>0</y>
             <width>21</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="QtGuiApplication.qrc">
             <normaloff>:/png/open.png</normaloff>:/png/open.png</iconset>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="pipelineNameLineEdit"/>
           </item>
           <item>
            <widget class="QToolButton" name="savePipelineToolButton">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="QtGuiApplication.qrc">
               <normaloff>:/png/save.png</normaloff>:/png/save.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="pipelineTree">
          <column>
           <property name="text">
            <string/>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>4</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>21</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <widget class="QToolButton" name="addStageToolButton">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>21</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="QtGuiApplication.qrc">
             <normaloff>:/png/add.png</normaloff>:/png/add.png</iconset>
           </property>
          </widget>
          <widget class="QToolButton" name="removeStageToolButton">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>0</y>
             <width>21</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="QtGuiApplication.qrc">
             <normaloff>:/png/delete.png</normaloff>:/png/delete.png</iconset>
           </property>
          </widget>
          <widget class="QToolButton" name="saveStageToolButton">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>0</y>
             <width>21</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="QtGuiApplication.qrc">
             <normaloff>:/png/save.png</normaloff>:/png/save.png</iconset>
           </property>
          </widget>
          <widget class="QToolButton" name="toSpvToolButton">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>0</y>
             <width>21</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="QtGuiApplication.qrc">
             <normaloff>:/png/spv.png</normaloff>:/png/spv.png</iconset>
           </property>
          </widget>
          <widget class="QToolButton" name="explorerStageFileToolButton">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>0</y>
             <width>21</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="QtGuiApplication.qrc">
             <normaloff>:/png/open.png</normaloff>:/png/open.png</iconset>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTabWidget" name="stageTabWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>3</verstretch>
            </sizepolicy>
           </property>
          </widget>
          <widget class="QTabWidget" name="bottomTabWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>947</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtGuiApplication.qrc"/>
 </resources>
 <connections/>
</ui>
